src/Framework/README.md

```diff
@@ -0,0 +1,25 @@
+# Framework
+
+Contains packages that provide reference assembles, default sets APIs for building ASP.NET Core applications, and assets used for self-contained deployments.
+
+## Description
+
+The following contains a description of each sub-directory in the `Framework` directory.
+
+- `App.Ref.Internal`: Empty project for identifying the non-stable version of AspNetCore at the time the targeting pack was last built
```

Comment:
May be worth mentioning the created package doesn't ship externally and shouldn't be important to 99.72% of developers